code/modules/antagonists/changeling/powers/headcrab.dm

```diff
@@ -37,13 +37,15 @@
 		user.investigate_log("has been gibbed by headslug burst.", INVESTIGATE_DEATHS)
 	user.gib()
 	. = TRUE
-	addtimer(CALLBACK(src, PROC_REF(spawn_headcrab), stored_mind, user_turf, organs), 3 SECONDS)
+	// So that the headslug does not get blown up, this is ugly
```

Comment:
This comment isn't really necessary, it is only providing historical context which you could instead just find via this PR.